tests/integration_tests/test_simple.py

```diff
@@ -0,0 +1,38 @@
+# This is an example test file. It won't get merged
+
+import pytest
+
+from tests.integration_tests.platforms import (
+    dynamic_client, OciClient, LxdContainerClient
+)
+
+
+CLOUD_CONFIG = """#cloud-config
+bootcmd:
+  - echo 'hello config!' > /tmp/cloud_config.txt"""
+
+
+class TestSimple:
+    @pytest.mark.cloud_config(CLOUD_CONFIG)
+    def test_cloud_config(self, client):
+        client.push_to_file('/home/ubuntu/my_test.txt', 'Hello world!')
+        print(client.exec('cloud-init -v'))
```

Comment:
Since these are tests, let's assert the heck outta this stuff instead of prints. We can leave the introductory prints in for documentation. But best to show a suggestion of how we expect these tests to look (a bit).

Reply:
Yeah, this is just an example file for playing around with concepts. I don't plan on merging it. I'll come up with some other examples too.